qiskit_machine_learning/runtime/torch_runtime_client.py

```diff
@@ -0,0 +1,513 @@
+        Always fails to initialize
+        """
+
+        def __init__(self) -> None:
+            raise MissingOptionalLibraryError(
+                libname="Pytorch",
+                name="TorchConnector",
+                pip_install="pip install 'qiskit-machine-learning[torch]'",
+            )
+
+
+class TorchRuntimeResult:
+    """The TorchRuntimeClient result object.
+
+    The result objects contains the state dictionary of the trained model,
+    and the training history such as the value of loss function in each epoch.
+    """
+
+    def __init__(self) -> None:
+        self._job_id: Optional[str] = None
+        self._train_history: Optional[Dict[str, Any]] = None
+        self._val_history: Optional[Dict[str, Any]] = None
+        self._model_state_dict: Optional[Dict[str, Any]] = None
+        self._train_time: Optional[float] = None
+
+    @property
+    def job_id(self) -> str:
+        """The job ID associated with the Torch runtime job."""
+        return self._job_id
+
+    @job_id.setter
+    def job_id(self, job_id: str) -> None:
+        """Set the job ID associated with the Torch runtime job."""
+        self._job_id = job_id
+
+    @property
+    def train_history(self) -> Dict[str, Any]:
+        """The train history"""
+        return self._train_history
+
+    @train_history.setter
+    def train_history(self, history: Dict[str, Any]) -> None:
+        """Set the train history"""
+        self._train_history = history
+
+    @property
+    def val_history(self) -> Dict[str, Any]:
+        """Returns validation history"""
+        return self._val_history
+
+    @val_history.setter
+    def val_history(self, history: Dict[str, Any]) -> None:
+        """Sets validation history"""
+        self._val_history = history
+
+    @property
+    def model_state_dict(self) -> Dict[str, Any]:
+        """Returns state dictionary of trained model"""
+        return self._model_state_dict
+
+    @model_state_dict.setter
+    def model_state_dict(self, state_dict: Dict[str, Any]) -> None:
+        """Sets state dictionary of trained model"""
+        self._model_state_dict = state_dict
+
+    @property
+    def train_time(self) -> float:
+        """Returns training time"""
+        return self._train_time
+
+    @train_time.setter
+    def train_time(self, time: float) -> None:
+        """Sets training time"""
+        self._train_time = time
+
+
+class TorchRuntimeClient:
+    """The Qiskit Machine Learning Torch runtime client to call the Torch runtime."""
+
+    def __init__(
+        self,
+        model: Module,
+        optimizer: Optimizer,
+        loss_func: Union[_Loss, Callable],
+        epochs: int = 10,
+        shots: int = 1024,
+        measurement_error_mitigation: bool = False,
+        provider: Optional[Provider] = None,
+        backend: Optional[Backend] = None,
+    ) -> None:
+        """
+        Args:
+            model: A PyTorch nn.Module to be trained.
```

Comment:
Or replace with ```torch.nn.Module``` in double quotes.
```suggestion
            model: A PyTorch module to be trained.
```